tests/test_trigger_out.cpp

```diff
@@ -0,0 +1,9 @@
+#include "sm_tests.h"
+
+/** @brief Fills [SWS_SM_00020] */
+TEST_F(smTests, testInternalStatePropagation) {
+    // let changes to be applied
```

Comment:
What changes? You didn't call SMSetState(on)

Reply:
It's no logic implemented yet in SM. Now, SM sets TriggerOut notifier without any conditions